x-pack/plugins/security_solution/server/endpoint/routes/metadata/index.ts

```diff
@@ -106,4 +107,23 @@ export function registerEndpointRoutes(
     },
     getMetadataRequestHandler(endpointAppContext, logger)
   );
+
+  router.get(
+    {
+      path: `/api/endpoint/toAgentID`,
```

Comment:
the path should be snake_cased instead of CamelCased
also - what is the porpose of this API?

Reply:
this... is not actually specifically needed anywhere. I used in during testing to validate the translation worked externally.

But I can't say it's not a very tempting thing to expose.